test/tools/runner/index.js

```diff
@@ -12,7 +12,17 @@ const { getEnvironmentalOptions } = require('../utils');
 const mock = require('../mongodb-mock/index');
 const { inspect } = require('util');
 
-const MONGODB_URI = process.env.MONGODB_URI || 'mongodb://localhost:27017';
+// Default our tests to have auth enabled
+process.env.AUTH = process.env.AUTH === 'noauth' ? 'noauth' : 'auth';
```

Comment:
I'm open to thoughts here.  With this change I wanted to

1. default tests to running with auth when running locally (or in CI)
2. not require changes to the scripts in package.json or require passing in environment variables

If anyone has a better thought about how to accomplish this, I'm open to suggestions

Reply:
It makes sense why you had to do this to make the auth setting globally available, I think we can defer to a better solution in NODE-3714, if you could reference that in a todo here.

Reply:
Done!